tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs

```diff
@@ -124,7 +124,12 @@ public static class TestConstants
         public const string KeyVaultReference = "AzureADIdentityDivisionTestAgentCert";
 
         // Integration tests
-        public const string SecurePageGetTokenAsync = "/SecurePage/GetTokenAsync";
+        public const string EmptyGetEmpty = "/Empty";
+        public const string TokenAcquisitionGetEmpty = "/TokenAcquisition";
+        public const string GraphClientGetEmpty = "/GraphClient";
+        public const string SecurePageGetEmpty = "/SecurePage/GetEmpty";
```

Comment:
nit: should this be "/GraphClient/GetEmpty" or change the variable name to something else?